workflow-job

```diff
@@ -573,7 +573,7 @@ def buildVolumeFromTag(def node) {
 def compile() {
     sh "docker build --tag=${images.jamesCompile} dockerfiles/compilation/java-8"
 
-    def tmpfsOptions = '--tmpfs=/tmp:rw,exec,size=1g --tmpfs=/james-project:rw,size=4g'
+    def tmpfsOptions = '--tmpfs=/tmp:rw,exec,size=1g --tmpfs=/root/.m2:rw,exec,size=1g --tmpfs=/james-project:rw,size=4g'
```

Comment:
what does `size=1g` mean? If it is the size limitation then I think you should increase, my .m2 directory takes above 2GB

Reply:
But your .m2 directory have not the scope of a single build... You have many version of many different dependencies, on many different projects.

Reply:
Total size is about 350 MO (local compilation), then 1 GO is faaar from enough